engines/scumm/scumm-md5.h

```diff
@@ -3,6 +3,12 @@
   DO NOT EDIT MANUALLY!
  */
 
+#ifndef SCUMM_MD5_INTERNAL_H
```

Comment:
For this you should edit `c_header` in `./devtools/md5table.cpp`

Reply:
Thanks, done.